src/acl/FilledChecklist.cc

```diff
@@ -218,9 +218,15 @@ ACLFilledChecklist::ACLFilledChecklist(const acl_access *A, HttpRequest *http_re
     rfc931[0] = '\0';
 
     changeAcl(A);
+    setRequest(http_request);
+    setIdent(ident);
+}
 
-    if (http_request != NULL) {
-        request = http_request;
+void ACLFilledChecklist::setRequest(HttpRequest *httpRequest)
+{
+    assert(!request);
+    if (httpRequest != NULL) {
```

Comment:
please do not leave NULL in altered code. Use if(!httpRequest) here instead.

Reply:
> Use if(!httpRequest) here instead.
I would say 'use if (httpRequest) here instead'
Done.